packages/babel-plugin-transform-es2015-block-scoping/test/fixtures/general/switch/actual.js

```diff
@@ -9,3 +11,15 @@ switch (a) {
     let d = 3;
     break;
 }
+
+switch (true) {
+  case true:
+    let a = false;
+    let b = true;
+    let c = 4;
+    let d = 5;
+
+  case false:
+    class e {}
+    function f() {}
+}
```

Comment:
I'm not familiar with this code, but does this cover cases like

```
    switch(true){
      case true:
        let a = 4;
      case false:
        a = 5;
        console.log(a);
    }
```
properly? We should make sure the tests cover that to be sure.